src/Loader.php

```diff
@@ -165,6 +165,10 @@ public static function user_can_analytics() {
 	 * @return bool Returns true if the feature is enabled.
 	 */
 	public static function is_feature_enabled( $feature ) {
+		if ( 'navigation' === $feature && 'yes' !== get_option( 'woocommerce_navigation_enabled', 'no' ) ) {
```

Comment:
I think this logic can be moved into `Navigation.php` and we can use the `woocommerce_admin_features` filter.  This allows us to continue using `Loader::is_feature_enabled()` to check for features.

See https://github.com/woocommerce/woocommerce-admin/blob/eb6ac6e9708bc4139a4cf11c1f92c703813ba222/src/FeaturePlugin.php#L204 for prior art.